### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by noelyoo**
This PR 
* updates Node.js versions for Travis CI build with the supported versions
* fixes https://github.com/indutny/bn.js/issues/204

image

Details: https://github.com/nodejs/Release#end-of-life-releases

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,14 @@
 sudo: false
 language: node_js
 node_js:
-  - "0.10"
-  - "0.12"
-  - "4"
-  - "5"
+  - "lts/*"
```
Comment (noelyoo) [inline .travis.yml]: I used `- "lts/*"` here @dcousens

**Comment by indutny** [comment]
Thanks for PR!

Should we also add `stable`?

**Comment by noelyoo** [comment]
@indutny Hi, regarding [this](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions), adding `node` will do I think.

**Comment by indutny** [comment]
Right! Sorry.

**Comment by indutny** [comment]
LGTM, thank you!

**Pull request merged**